### PR TITLE
Use an external database for conduktor

### PR DESCRIPTION
**Pull request by rokh-conduktor**

### CODE REVIEW

File: conduktor.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      POSTGRES_HOST_AUTH_METHOD: "scram-sha-256"
+
+  conduktor-platform:
+    image: conduktor/conduktor-platform:latest
+    ports:
+      - "8080:8080"
+    volumes:
+      - conduktor_data:/var/conduktor
+    depends_on:
+      - kafka1
```
Comment (AlexandreBrg) [inline conduktor.yml:21]: The dependency kafka1 is not in this docker compose 😉

File: conduktor.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      CDK_ORGANIZATION_NAME: "demo"
+      CDK_ADMIN_EMAIL: "admin@admin.io"
+      CDK_ADMIN_PASSWORD: "admin"
+      CDK_DATABASE_URL: "postgresql://conduktor:some_password@postgresql:5432/conduktor-platform"
+      CDK_CLUSTERS_0_ID: "default"
+      CDK_CLUSTERS_0_NAME: "My Local Kafka Cluster"
+      CDK_CLUSTERS_0_COLOR: "#0013E7"
+      CDK_CLUSTERS_0_BOOTSTRAPSERVERS: "PLAINTEXT://kafka1:19092"
+      CDK_CLUSTERS_0_SCHEMAREGISTRY_URL: "http://kafka-schema-registry:8081"
+      CDK_CLUSTERS_0_KAFKACONNECTS_0_URL: "http://kafka-connect:8083"
```
Comment (AlexandreBrg) [inline conduktor.yml:32]: I believe those piece of variables should be described in each docker compose, as in some case you have a schema registry while in some other you don't. 

Environment variables from the docker composed that extends are added above env var given in the extended docker compose, so no worries, it shouldn't break anything ;D 

(`extends` [docs](https://docs.docker.com/compose/extends/#extending-services))

File: conduktor.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  postgresql:
+    image: postgres:14
+    hostname: postgresql
+    environment:
+      POSTGRES_DB: "conduktor-platform"
+      POSTGRES_USER: "conduktor"
+      POSTGRES_PASSWORD: "some_password"
+      POSTGRES_HOST_AUTH_METHOD: "scram-sha-256"
+
+  conduktor-platform:
```
Comment (AlexandreBrg) [inline conduktor.yml:14]: Consider adding a health check to be sure to wait for platform to be healthy:

```yaml
    healthcheck:
      test: curl -f http://localhost:8080/platform/api/modules/health/live || exit 1
      interval: 10s
      start_period: 10s
      timeout: 5s
      retries: 3
```

**Review by AlexandreBrg — COMMENTED** [review]
🥳

File: conduktor.yml (hunk 1 above)
Comment (rokh-conduktor) [reply conduktor.yml:21]: it is in all the place where this one is used :)

File: conduktor.yml (hunk 2 above)
Comment (rokh-conduktor) [reply conduktor.yml:32]: not critical, it just set the config, if no Kconnect, Kconnect features will not work, that's all.

**Comment by rokh-conduktor** [comment]
linked to : https://github.com/conduktor/conduktor-platform/pull/147

File: conduktor.yml (hunk 2 above)
Comment (qboileau) [reply conduktor.yml:36]: Yest but it shows the configuration alongside the other services. Also, why configure something that we know will not work ? 
Those composes file are here to explain how to setup kafka and Conduktor platform together in different configuration. Showing that in each docker-compose is important. 
You can still keep conduktor.yaml with platform service for global configuration.

**Review by qboileau — COMMENTED** [review]
Same remarks as @AlexandreBrg

File: conduktor.yml (hunk 1 above)
Comment (rokh-conduktor) [reply conduktor.yml]: already removed

File: conduktor.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+version: '3.8'
+
+services:  
+  
+  postgresql:
+    image: postgres:14
+    hostname: postgresql
```
Comment (qboileau) [inline conduktor.yml:7]:
```suggestion
    hostname: postgresql
    volumes:
      - pg_data:/var/lib/postgresql/data
```

**Pull request merged**